portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,6 +37,16 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    int maxComment = 0;  
+    String maxCommentstring = request.getParameter("max-comment");
+
+    try {
+      maxComment = Integer.parseInt(maxCommentstring);
+    } catch (NumberFormatException e) {
+      System.err.println("Could not convert to int: " + maxCommentstring);
```

Comment:
Nit: You could consider saying which string you couldn't convert, i.e., something like, "Could not convert max comment to int(...)"